config/config.exs

```diff
@@ -62,6 +62,10 @@ config :mime, :types, %{
   "application/vnd.hex+erlang" => ["erlang"]
 }
 
+config :mime, :extensions, %{
+  "json" => "application/json"
+}
```

Comment:
What is this needed for?

Reply:
when i was getting everything setup on my machine and compiling deps mime showed a compiler warning for this. I just removed it and it seems fine now.